packages/teuchos/comm/src/Teuchos_TimeMonitor.cpp

```diff
@@ -260,7 +260,7 @@ namespace Teuchos {
       counter().start(reset);
 #ifdef HAVE_TEUCHOS_ADD_TIME_MONITOR_TO_STACKED_TIMER
       if (nonnull(stackedTimer_))
-        stackedTimer_->start(counter().name());
+        stackedTimer_->start(counter().name(),false);
```

Comment:
So, if you use teuchos you don't get the kokkos injection but you do if you use stacked timer directly?  Should they be the same?

Reply:
This prevents double injection into kokkos. The TimeMonitor timers inject into the kokkos profiler inside the Time object. But if TimeMonitor injects data into StackedTimer as well, we were doubly injecting into kokkos.